iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -81,13 +84,15 @@ private static void writeModel(ShuntCompensator sc, NetworkXmlWriterContext cont
             XmlUtil.writeDouble("gPerSection", sc.getModel(ShuntCompensatorLinearModel.class).getGPerSection(), context.getWriter());
             context.getWriter().writeAttribute(MAXIMUM_SECTION_COUNT, Integer.toString(sc.getMaximumSectionCount()));
         } else if (sc.getModelType() == ShuntCompensatorModelType.NON_LINEAR) {
-            context.getWriter().writeStartElement(context.getVersion().getNamespaceURI(), SHUNT_NON_LINEAR_MODEL);
-            for (ShuntCompensatorNonLinearModel.Section s : sc.getModel(ShuntCompensatorNonLinearModel.class).getAllSections()) {
-                context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), "section");
-                XmlUtil.writeDouble("b", s.getB(), context.getWriter());
-                XmlUtil.writeDouble("g", s.getG(), context.getWriter());
-            }
-            context.getWriter().writeEndElement();
+            IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_3, context, () -> {
+                context.getWriter().writeStartElement(context.getVersion().getNamespaceURI(), SHUNT_NON_LINEAR_MODEL);
+                for (ShuntCompensatorNonLinearModel.Section s : sc.getModel(ShuntCompensatorNonLinearModel.class).getAllSections()) {
+                    context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), "section");
+                    XmlUtil.writeDouble("b", s.getB(), context.getWriter());
+                    XmlUtil.writeDouble("g", s.getG(), context.getWriter());
+                }
+                context.getWriter().writeEndElement();
+            });
```

Comment:
If we encounter a non linear model, and should export it in a previous version, we silently ignore it. Should we throw an exception? The exported network is not equivalent then.

Reply:
No, we don't, a log is made (l.51) but it is considered a shunt with a unique section which has the current susceptance